### DISCUSSION TITLE
SPA uygulaması yapmak ve Github Page için Actions deployment

### DISCUSSION THREAD

**Discussion by AimTune** [thecoderverse/community#4] [Ideas]
SPA uygulaması yapmak ve Github Page için Actions deployment

**Comment by buraksenyurt** [comment]
Bu ikisi farklı uygulama pratikleri değil mi? Yani iki ayrı çalışmadan bahsediyoruz. [MiniVerse](https://github.com/futureverseofficial/MiniVerse) altında plana dahil edebiliriz diye düşünüyorum.

**Reply by AimTune** [reply]
Evet. Ayrı sessionlarda anlatılabilecek çalışmalar. İkisini de [MiniVerse](https://github.com/futureverseofficial/MiniVerse) içerisine dahil ediyorum.
